rsopt/util.py

```diff
@@ -121,4 +123,26 @@ def _libe_save(H, persis_info, mess, filename):
     np.save(filename, H)
 
     with open(filename + ".pickle", "wb") as f:
-        pickle.dump(persis_info, f)
\ No newline at end of file
+        pickle.dump(persis_info, f)
+
+
+def get_objective_function(import_list: typing.List[str]) -> callable:
+    """Returns the function object from module.
+
+    Args:
+        import_list: (list) [path to module (str), function name (str)]
+
+    Returns: (callable) function
+
+    """
+
+    # import the objective function if given
+    if len(import_list) == 2:
+        module_path, function = import_list
+        sys.path.append(os.getcwd())
+        module = pkrunpy.run_path_as_module(module_path)
+        function = getattr(module, function)
+    else:
+        function = None
```

Comment:
What if import_list is just bad? Do we want an assert in here instead of just an else?